_posts/2023-10-06-utah-sgid-statewide-roads-data-layer-updates-10062023.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates 10/06/2023
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2023-10-06 8:30:00
+categories:
+  - Featured
+tags:
+  - roads
+---
+
+UGRC recently completed updates to the SGID Roads data layer. Please visit our [Roads and Highway System]({% link data/transportation/roads-system/index.html %}) data page where you will find information about the Roads data model, as well as a web service layer to the SGID Roads data and direct download links in shapefile and geodatabase format.
+
+These updates are also reflected in UGRC's [address locators]({% link data/address-geocoders-locators/index.html %}), Utah's [Next-Generation 911 GIS database]({% link data/911/index.html %}), and UGRC's [street network analysis datasets]({% link data/transportation/street-network-analysis/index.html %}).
+
+The following are highlights from this month's update.
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Davis County:** Obtained roads data on 09/21/2023. Previous update was on 09/08/2023.
+- **Iron County:** Obtained roads data on 09/13/2023. Previous update was on 09/07/2022.
```

Comment:
**quibble**: was iron really from 2022?

Reply:
yeah, crazy enough we still have a few counties on an annual update.  that's one of the things i'd like to improve/solve with portal and branch versioning.